workspace/api/openfdd_bridge/routes/bacnet_routes.py

```
@@ -418,12 +449,34 @@ def bacnet_trigger_poll() -> dict:
+    # Same-host Docker publish / LAN edge — not exposed on WAN when Caddy only fronts :80
+    return host.startswith(("172.", "10.", "192.168."))
+
+
+def _run_ingest_background() -> None:
+    if not _ingest_lock.acquire(blocking=False):
+        return
+    try:
+        result = ingest_poll_samples_to_feather()
+        if not result.get("ok"):
+            _log.warning("bacnet poll ingest: %s", result.get("reason") or result)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)